docs/content/topics/project-developers/projects.md

```diff
@@ -9,296 +9,267 @@ aliases:
+    iii. The `id`, or name, of the Project
+    iv. A `description` of the project
+  2. The `eventSubscriptions` configuration, which contains one event source
+    (`brigade.sh/cli`) and one type under that source (`exec`). This particular
+    configuration corresponds to events that arise from `brig event create`
+    commands.
+  3. The `workerTemplate`, which represents the configuration for the worker in
+    charge of running the script associated with this project. This particular
+    configuration has `logLevel` set to `DEBUG` and the `brigade.js` script for
+    this project defined in-line under `defaultConfigFiles`. We'll discuss
+    similar scripts in more detail in the [Scripting Guide], but for now we see
+    that this script imports the `events` object from the brigadier library and
+    declares an event handler for the event source/type combination mentioned
+    above.  In response to such events, it prints "Hello, World!" to the
+    console.
+
+For further examples of project definition files to help you get started, see
+the [examples directory].  Each example sub-directory will have a
+`project.yaml` file - this is the project definition file with configuration to
+that specific project.
+
+[examples directory]: ./examples
+[Scripting Guide]: /topics/scripting
+
+### Brig init
+
+To quickly bootstrap a new project, Brigade offers the `brig init` command. It
+will generate a new project definition file based on the options provided,
+which can then be used to create the project in Brigade.
+
+For example, to initialize a project named `myproject` with default settings,
+which includes TypeScript as the scripting language and no git configuration,
+run the following:
 
-This same pattern can be used to store a default `brigade.json` configuration file for a project.  The associated prompt is *Upload a default brigade.json config*.
+```console
+$ brig init --id myproject
+```
 
-### Managing Your Projects
+Or, if the alternate scripting language option of JavaScript is preferred, run:
 
-With `brig project create`, your projects are stored in Kubernetes secrets. You
-can save a local copy of that secret using `brigade project create --out=myproject.json`.
+```console
+$ brig init --id myproject --language js
+```
 
-If you have already created the secret, you can fetch it from Kubernetes by running
-`brig project get my/project` where `my/project` is the project name you assigned.
+If the project is git-based, supply the git repository name where the Brigade
+script for this project will reside:
 
-## Creating and Managing a Project (The Old Way)
+```console
+$ brig init --id myproject --git https://github.com/<org>/<repo>.git
+```
 
-Note: Managing Brigade projects via Helm chart is being deprecated in favor of using `brig`.
+A few assets will be created in the directory in which the command is run.
+The bulk of the generated files can be found in the `.brigade` directory,
```

Comment:
Nit: A trailing forward slash (`.brigade/`) often adds visual clarity that we're discussing a directory.